wiki/osu!stream/en.md

```diff
@@ -1,15 +1,15 @@
-<!-- TODO: some updates and needs a proper end of life message. it's also open source now -->
-
 # osu!stream
 
 ![](img/Os-Logo.jpg "osu!stream logo")
 
-**osu!stream** is a version of osu! built from scratch that is made especially for touch-enabled devices. It is currently available for Apple's [iOS](http://en.wikipedia.org/wiki/IOS) platform only, but an [Android](http://en.wikipedia.org/wiki/Android_%28operating_system%29) port is planned. It introduces a brand new game-play system such as the hold circle and connected beats.
+**osu!stream** is a version of osu! built from scratch that is made especially for touch-enabled devices. It is currently available for Apple's [iOS](http://en.wikipedia.org/wiki/IOS) platform only. The development ended in 2020 and is to be seen as finished. An [Android](http://en.wikipedia.org/wiki/Android_%28operating_system%29) port was planned but not implemented yet. For more information, read the [blog post](https://blog.ppy.sh/osu-stream-2020-release/) about its last release.
```

Comment:
Android support is in a working state these days, I need to do a final review pass on it but it will likely be available this year.

Reply:
Can be mentioned/announced later